test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -877,7 +877,7 @@ switch ($x)
                 @{ inputStr = '[math].G'; expected = 'GenericParameterAttributes'; setup = $null }
                 @{ inputStr = '[Environment+specialfolder]::App'; expected = 'ApplicationData'; setup = $null }
                 @{ inputStr = 'icm {get-pro'; expected = 'Get-Process'; setup = $null }
-                @{ inputStr = 'write-ouput (get-pro'; expected = 'Get-Process'; setup = $null }
+                @{ inputStr = 'write-output (get-pro'; expected = 'Get-Process'; setup = $null }
```

Comment:
I wonder the test works!

Reply:
> I wonder the test works!

Thanks for your review. I think there should be no problem with this modification, but I didn't test it.